ISISComputingGroup/EPICS-sampleChanger
# Ticket6409 select any position via script

See https://github.com/ISISComputingGroup/IBEX/issues/6409

(Note: Contains changes from https://github.com/ISISComputingGroup/IBEX/issues/6392 so merge that first)

To test:
* Start IOC under IOC_test_framework
* Add a block on to `LKUP:SAMPLE:POSN`
* Select a rack on the OPI
* Set the block to a position on a different rack
* Confirm starts moving to this position and that the rack changes accordingly
